charts/prefect-worker/README.md.gotmpl

```diff
@@ -213,6 +213,14 @@ prefect work-pool get-default-base-job-template --type kubernetes > base-job-tem
 helm install prefect-worker prefect/prefect-worker -f values.yaml --set-file worker.config.baseJobTemplate.configuration=base-job-template.json
 ```
 
+#### Updating the Base Job Template
```

Comment:
I think this being removed in https://github.com/PrefectHQ/prefect-helm/pull/368 was unintentional, so let's add it back for now. Can remove it later if we really don't want it.